routing.rst

```diff
@@ -843,87 +922,941 @@ You can also use special attributes to configure them (except ``_fragment``):
             ;
         };
 
-These attributes can also be used for route imports.
-
 .. versionadded:: 4.3
 
     The special attributes were introduced in Symfony 4.3.
 
-.. _routing-trailing-slash-redirection:
+Extra Parameters
```

Comment:
I know this is probably more the dev-me than the doc-me, but these are actually no longer parameters, but route attributes. Is it maybe important to differentiate between them?

Reply:
Originally we had a section called "route attributes" ... but we removed it, because we now explain "request attributes" (in this section -> `Getting the Route Name and Parameters`).

I wouldn't introduce more terms because everything is already packed with terms.